test/e2e/resource_generators.go

```diff
@@ -69,16 +70,51 @@ type azureConfig struct {
 }
 
 var (
-	// TODO Parameterize some of these variables
-	location       = GetRegion()
-	vmSize         = "Standard_D2s_v3"
-	namespace      = "default"
-	imageOffer     = "capi"
-	imagePublisher = "cncf-upstream"
-	imageSKU       = "k8s-1dot16dot6-ubuntu-1804"
-	imageVersion   = "latest"
+	LOCATION        string
```

Comment:
I don't think we need to capitalize the var names here, viper should be able to detect that the environment variable `LOCATION` maps to the golang var `location`, https://github.com/spf13/viper#env-example.

Reply:
done